app/assets/stylesheets/modules/results.css

```diff
@@ -104,3 +104,23 @@
     list-style-type: none;
   }
 }
+
+#modules {
+  flex-wrap: wrap;
+  gap: 3rem;
+}
+
+#modules aside, #modules .main-contents {
+  display: contents;
```

Comment:
Where is this from? I don't see it https://developer.mozilla.org/en-US/docs/Web/CSS/display

Reply:
There's a lot of discussion about the limitations out there. My understanding that note isn't accurate for modern browser versions. It might have been true 5 years ago, but the current state seems to be:
- it's probably a good thing the element isn't announced as an aside (if we've in-lined the contents into the parent)
- the point about descendant elements seems to be for IE and older versions of safari
- there's still some weird behavior for buttons (per caniuse)